src/components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Container, Title } from './VideoCard.styles';
+
+const VideoCard = ({ video, handleVideoSelected }) => {
```

Comment:
`handleVideoSelected` could be abstracted into the context and be used inside the component using a hook.